.evg.yml

```diff
@@ -175,15 +175,14 @@ functions:
           MDBJDBC_VER: "$MDBJDBC_VER"
           JAVA_HOME: "$JAVA_HOME"
           PREPARE_SHELL: |
-            export ADL_TEST_LOCAL_USER=${adl_test_local_user}
-            export ADL_TEST_LOCAL_PWD=${adl_test_local_pwd}
-            export ADL_TEST_LOCAL_HOST=${adl_test_local_host}
```

Comment:
This variable is not used anywhere in the repo and the evergreen variable was never defined, so I removed it here.